app/serializers/site_serializer.rb

```diff
@@ -110,17 +110,49 @@ def groups
     end
   end
 
+  def like_action_type
+    # Like is very special post action type which cannot be disabled or edited in admin panel
+    Flag.new(
+      id: PostActionType::LIKE_POST_ACTION_ID,
+      name: "like",
+      name_key: "like",
+      description: "",
+      position: 0,
+    )
```

Comment:
Is there a reason we don't just query this record from the database instead of creating the record ourselves here?

Reply:
Like is a bit of a snowflake. It is more like PostActionType than Flag. It cannot be disabled, edited, or moved. I can add it to the database and filter UI to not show it.